keras_nlp/models/albert/albert_classifier.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ALBERT classification model."""
+import copy
+
+from tensorflow import keras
+
+from keras_nlp.models.albert.albert_backbone import albert_kernel_initializer
+from keras_nlp.models.task import Task
+from keras_nlp.utils.python_utils import classproperty
+from keras_nlp.models.albert.albert_presets import backbone_presets
+from keras_nlp.models.albert.albert_backbone import AlbertBackbone
+from keras_nlp.models.albert.albert_preprocessor import AlbertPreprocessor
+
+
+@keras.utils.register_keras_serializable(package="keras_nlp")
+class AlbertClassifier(Task):
+    """An end-to-end ALBERT model for classification tasks
+
+    This model attaches a classification head to a `keras_nlp.model.AlbertBackbone`
+    backbone, mapping from the backbone outputs to logit output suitable for
+    a classification task. For usage of this model with pre-trained weights, see
+    the `from_preset()` method.
+
+    This model can optionally be configured with a `preprocessor` layer, in
+    which case it will automatically apply preprocessing to raw inputs during
+    `fit()`, `predict()`, and `evaluate()`. This is done by default when
+    creating the model with `from_preset()`.
+
+    Disclaimer: Pre-trained models are provided on an "as is" basis, without
+    warranties or conditions of any kind.
+
+    Args:
+        backbone: A `keras_nlp.models.AlertBackbone` instance.
+        num_classes: int. Number of classes to predict.
+        dropout: float. The dropout probability value, applied after the dense
+            layer.
+        preprocessor: A `keras_nlp.models.AlbertPreprocessor` or `None`. If
+            `None`, this model will not apply preprocessing, and inputs should
+            be preprocessed before calling the model.
+
+    Examples:
+
+    ```python
+    # Call classifier on the inputs.
```

Comment:
We will need to add some more documentation for overall usages here, see BERT as an example -> https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/bert/bert_classifier.py#L55-L161

But I'm also OK with this being a follow up, we have some work to do on our class level docstrings anyway.

Reply:
Let's just do this one as a follow up, I've opened https://github.com/keras-team/keras-nlp/issues/688. Other comments still apply!

Reply:
I made current docstrings alone with Bert classifier, once the pr merges and we have clear cut idea on how the docstrings for all models are to be restructured, I can put a follow up PR.